.github/workflows/silly_sprites.yml

```
@@ -16,9 +15,13 @@ jobs:
                 build: [ Release, Debug ]
         
         steps:
-        -   name: "checkout"
+        -   name: "Checkout"
             uses: actions/checkout@v3
             
+        -   name: "Install apt for Ubuntu"
```

`apt` is not what's being installed here, it's `xorg-dev`
```suggestion
        -   name: "Install xorg-dev for Ubuntu"
```